ppg-server/ppg-server.spec

```diff
@@ -0,0 +1,30 @@
+%global sname   percona-ppg-server
+%global pgmajorversion 14
+%global version 4
```

Comment:
We should start with version 0. There is no code change as such, only packaging related changes.